### ISSUE TITLE
Essentials /repair doesn't work

### ISSUE THREAD

**Issue by tonnic** [keepcalm/BukkitForge#550]
Using BukkitForge-1.4.7-252.jar.
2.10.1 version of Essentials.

Also a minor thing, in Towny, the town warps do not take any money to do that.

**Comment by Ionaru** [comment]
Confirmed, repairing tools does not work.

**Comment by eagl3s1ght** [comment]
Please check if there's any error when using /repair after using the **/ess debug command**. This may make it easier for the developers to solve the problem without having to reproduce the error.

**Comment by tonnic** [comment]
Nope... no error.  It says that the item was successfully repaired, but it doesn't do anything.

**Comment by zuch0698o** [comment]
Can confirm replicated the issue on dev realm and public server however with custom npcs you can still acompkish it the repair guy role doesnt seem affected

**Comment by tonnic** [comment]
Custom npcs?  Can you charge for that with essentials economy?
